logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -61,7 +61,6 @@
     private final int maxUnread;
     private final int checkpointMaxAcks;
     private final int checkpointMaxWrites;
-    private final int checkpointMaxInterval;
```

Comment:
@robbavey my guess too, I think this is still being debated even, but as long as it's not functional I see no point in keeping it around and making things even more complicated than they already are :)